internal/venice-common/src/main/java/com/linkedin/venice/views/MaterializedView.java

```diff
@@ -0,0 +1,57 @@
+package com.linkedin.venice.views;
+
+import com.linkedin.venice.exceptions.VeniceException;
+import com.linkedin.venice.meta.Store;
+import com.linkedin.venice.meta.Version;
+import com.linkedin.venice.meta.ViewParameterKeys;
+import com.linkedin.venice.utils.VeniceProperties;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Properties;
+
+
+public class MaterializedView extends VeniceView {
+  public static final String MATERIALIZED_VIEW_TOPIC_SUFFIX = "_mv";
+  private static final String MISSING_PARAMETER_MESSAGE = "%s is required for re-partition view!";
+
+  public MaterializedView(Properties props, Store store, Map<String, String> viewParameters) {
+    super(props, store, viewParameters);
+  }
+
+  @Override
+  public Map<String, VeniceProperties> getTopicNamesAndConfigsForVersion(int version) {
```

Comment:
I understand this is an existing API, but the returned object seems to be a `Pair` rather a map as it would only contain one entry, right?

Reply:
Correct, I'm guessing that the original intention is that some view could have multiple topics?

Reply:
Really?
@ZacAttack Do you know such use case?

Reply:
To speed up the review, and we can ignore this comment for now as it is not a big issue and we can refactor it later if needed.